concordium-sdk/src/main/java/com/concordium/sdk/responses/branch/Branch.java

```diff
@@ -0,0 +1,38 @@
+package com.concordium.sdk.responses.branch;
+
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.Hash;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.*;
+
+import java.util.List;
+
+@Data
+@ToString
+@RequiredArgsConstructor
+public class Branch {
```

Comment:
Please doc this. 
E.g. "The branches above the last finalized block."

Reply:
done